packages/mainsail/source/networks/ark.devnet.ts

```diff
@@ -25,6 +25,10 @@ const network: Networks.NetworkManifest = {
 			host: "https://dwallets.mainsailhq.com/api",
 			type: "full",
 		},
+		{
```

Comment:
while at it, it would be best to update the configuration early on as well:
- rename it to `mainsail.devnet.ts` and `mainsail.mainnet.ts`
- leave mainnet untouched for now as the specifics for that one are unclear atm
- for devnet, adjust the data based on the https://dwallets.mainsailhq.com/api/node/configuration endpoint (things like epoch, amount of delegates, etc) (note: ignore the validator names and find the delegate version of it in the network config for now until we change it entirely to validators for mainsail)

Reply:
will take a look tomorrow because that implies adjusting networks name on manifest and so on

Reply:
will change it to a card, avoids blocking the rest of this PR